server/src/fty-srr-cmd.cc

```diff
@@ -182,7 +191,8 @@ int main (int argc, char **argv)
         } else {
             std::cout << "### - No input file specified, waiting for input from stdin" << std::endl;
         }
-        opRestore(passphrase, sessionToken, inputFile.is_open() ? inputFile : std::cin, force);
+        std::string reauthToken = buildReauthToken(sessionToken, reauthPasswd);
```

Comment:
We need to make sure password is correct before to processed to the restore (same behavior as UI)

Reply:
It's an option, but another one is: let usm srr do the check !

Reply:
No way, letting usm do the check is not acceptable solution. USM is the last module to be called, I do not want to have a partial restore because an user made a typo in the password. Please fix it.

Reply:
Done